profile-page.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Profilepage</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="profile-page.css" rel="stylesheet" type="text/css"> 
+       </div>
+    
+    </head>
+
+<body>
+    <div class="picture">
```

Comment:
Could you use other semantic html tags other than divs - `div` should be last resort